lib/assets/chart_builder/main.js

```diff
@@ -166,8 +171,12 @@ export function init(ctx, payload) {
     }
   }
 
-  function renderCurrentData(fields) {
-    disableAll();
+  function renderCurrentData(fields, dataOptions) {
+    const dataVariable = fields["data_variable"];
+    if (!dataOptions[dataVariable]) {
+      disableAll();
+    }
```

Comment:
This is the relevant part.

The remaining diff renaming `options` to explicitly be either data or axis options, as otherwise it's quite confusing which is which :cat2:

Reply:
Thanks 😊